common-practices-tools/security/README.md

```diff
@@ -172,7 +172,6 @@ When you delete a file, it doesn't actually go away. Usually, all that occurs is
 GNU/Linux:
 
 -   [How to delete file(s) in secure manner?](https://askubuntu.com/questions/57572/how-to-delete-files-in-secure-manner) (Ask Ubuntu)
--   [How to: Delete your Data Securely on Linux](https://ssd.eff.org/en/module/how-delete-your-data-securely-linux) (from the EFF Surveillance Self-Defense guide)
```

Comment:
@openprivacy instead of deleting do you have a replacement link? @mgifford suggested https://ostechnix.com/securely-permanently-delete-data-linux/

Reply:
Mike's link works - I'm good with that. And we may want to add that destroying the drive before throwing it away is another option.

Reply:
Updated and added a small note in the above paragraph about destroying

Reply:
Spulling eror: should be "permanently"

Reply:
Doh! Fixed it. Thank you

Reply:
I am pretty sure no-one on the team has HDDs on their work machine, so I think using  is bad advice. From the article:
> Also, all of these are unnecessary and dangerous on SSD drives

SSDs have their own built in secure erase function (NVMe/ATA Secure Erase) and this is really the only way you can be assured they are erased without involving a hammer.

I would suggest something like these processes look like a good start:
- MacOS: https://support.apple.com/guide/mac-help/erase-your-mac-mchl7676b710/mac or erase via macOS recovery works.
- Windows/Linux: https://www.tomshardware.com/how-to/secure-erase-ssd-or-hard-drive

Reply:
Updated the links